portal-web/test/functional/com/liferay/portalweb/tests/DummyDemo.testcase

```diff
@@ -0,0 +1,61 @@
+definition {
+
+	property testray.main.component.name = "Blogs";
+
+	setUp {
+		User.firstLoginUI();
+	}
+
+	test AddBlogsEntry {
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Blogs");
+
+		Blogs.addEntry(
+			entryContent = "This is the test content, for it to be saved as draft.",
+			entryTitle = "Test title");
+	}
+
+	test DeleteNewEntry {
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Blogs");
+
+		Blogs.addEntry(
+			entryContent = "Doesn't really matter.",
+			entryTitle = "To be deleted");
+
+		BlogsNavigator.moveToRecycleBinCP(entryTitle = "To be deleted");
+
+		ProductMenu.gotoPortlet(
+			category = "Recycle Bin",
+			portlet = "Recycle Bin");
+
+		RecycleBin.deleteCP(
+			assetName = "To be deleted",
+			assetType = "Blogs Entry");
+	}
+
+	test ModifyNewEntry {
```

Comment:
Please add proper description tag to the test.

Reply:
:)